src/main/scala/chisel3/util/experimental/package.scala

```diff
@@ -0,0 +1,34 @@
+package chisel3.util
+
+import chisel3.Module
+
+package object experimental {
```

Comment:
I'm all for just dropping this in the `chisel3` package. This should be, by default available if you do `import chisel3._`.

Reply:
Got it:) Let's skip the experimental

Reply:
Oh, that's another problem to drop it to chisel3 package:
`package object chisel3` is under `core` project, which doesn't depend on `chisel3` project:
So basically from current build architecture, `chisel3._` has no access to `stage`

Reply:
How about move it to `chisel3.stage` package? I think that's a good place for this API to stay.
I have no I idea how to add this API to chisel3 package.

Reply:
Why is this splitting between ```core``` and the other project in the first place? This is why I stumbled over not being able to implement ```emitVerilog()```  just as a method on ```RawModule```. A method on ```RawModule``` would be easier to read than using this implicit magic ;-)

Reply:
Yes, I also think `RawModule` is much reasonable.
I also wonder why splitting chisel3 into different packages, I think this is the question is to @jackkoenig